main.js

```diff
@@ -29,6 +29,7 @@ module.exports = function(options) {
 		withFlags: false,
 		withHandlebars: false,
 		withNavigation: false,
+		withNavigationHierarchy: false,
```

Comment:
Yeah, spent some time wondering about this (and googled a definition of taxonomy), hierarchy seems to make more sense here?